locustfile.py.example

```diff
@@ -0,0 +1,92 @@
+import logging
```

Comment:
Again - we're not looking at this with heavy scrutiny yet (or ever really - adjust this to your needs - this is just what I'm using this time around). But feel free to take a look and check out the output as things run (yes, 2 endpoints are angry and I'm still deciding what to do about it)